src/views/OrderDetail.vue

```diff
@@ -117,6 +117,7 @@ export default defineComponent({
   computed: {
     ...mapGetters({
       order: "order/getCurrent",
+      currentFacilityId: 'user/getCurrentFacility',
```

Comment:
We are storing entire facility object, not the facilityId. Variable names should be self explanatory
```suggestion
      currentFacility: 'user/getCurrentFacility',
```

Reply:
Changed getter name.